concent_api/core/views.py

```diff
@@ -3,19 +3,51 @@
 from django.views.decorators.http   import require_POST
 from django.views.decorators.http   import require_GET
 
+from golem_messages.exceptions      import FieldError
+from golem_messages.exceptions      import InvalidSignature
+from golem_messages.exceptions      import MessageError
+from golem_messages.exceptions      import MessageFromFutureError
+from golem_messages.exceptions      import MessageTooOldError
+from golem_messages.exceptions      import TimestampError
+
 from core.message_handlers          import handle_message
 from core.message_handlers          import handle_messages_from_database
 from core.subtask_helpers           import update_timed_out_subtasks
 
 from utils                          import logging
 from utils.api_view                 import api_view
 from utils.helpers                  import decode_client_public_key
+from utils.shortcuts                import load_without_public_key
 from .models                        import PendingResponse
 
 
 @api_view
 @require_POST
 def send(request, client_message):
+    if len(request.body) == 0:
```

Comment:
please extract it to a functi0on `validate_request` or sth similar and move it to `validations` module

Reply:
This piece of code was removed from here and will be back in `api_view` decorator.